service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AisConsentCreationSteps.java

```diff
@@ -64,7 +64,8 @@
 
   @Before
   public void setScaApproach() {
-    scaApproach = aspspProfileService.getScaApproach().toString();
+    //TODO how handle multiple SCA approaches?
```

Comment:
I would delete this howto. We just support one sca till our team commits to support multiple. Up to this moment, this solution is fine.